src/components/atoms/dropdown/modal.tsx

```diff
@@ -137,11 +130,20 @@ export const DropdownModal: React.FC<DropdownModalProps> = ({
         enabled>
         <ScrollView keyboardShouldPersistTaps="always">
           {items.map(renderItem)}
+          {!!search?.term && !searchHasResults && (
+            <View accessibilityElementsHidden style={listStyles.contentWrapper}>
+              <Text style={listStyles.noResults}>{search?.noResults}</Text>
+            </View>
```

Comment:
Moving this here shows the "no results" message if there is what we consider to be no results (e.g. on counties, if there is only "All counties"), and shows it under any such default results (i.e. under "All counties")